### PR TITLE
no message

### PR DESCRIPTION
**Pull request by KimIdeas8**

### CODE REVIEW

File: docs/team/kimideas8.md [hunk 1]
```diff
@@ -18,14 +18,14 @@ This application was created with the aim of helping NUS Computing and Engineeri
     - What it does: allows the user to add notes one at a time to a location of choice, view an existing list of notes by locations, and delete notes at a location of choice one at a time 
     - Justification: It provides users with way of making quick notes within the campus, and saves users the trouble of adding notes in their own note-taking softwares, especially if the user is rushing for time.
       Since it helps to naturally organise a person's notes by locations, the user can easily recall what the note was made for.
-      
+
+<div style="page-break-after: always;"></div>
```
Comment (wjchoi0712) [inline docs/team/kimideas8.md:22]: what does this do?

**Review by wjchoi0712 — APPROVED** [review]
Nice

File: docs/team/kimideas8.md (hunk 1 above)
Comment (KimIdeas8) [reply docs/team/kimideas8.md:22]: ohhhh its a page break for the conversion to pdf file in github.io site

Comment (wjchoi0712) [reply docs/team/kimideas8.md:22]: Oh then don't we need this for the UG and DG as well?

**Pull request merged**